src/Symfony/Component/HttpClient/RetryableHttpClient.php

```diff
@@ -28,7 +28,7 @@
  *
  * @author Jérémy Derussé <jeremy@derusse.com>
  */
-class RetryableHttpClient implements HttpClientInterface, LoggerAwareInterface, ResetInterface
+class RetryableHttpClient implements HttpClientInterface, ResetInterface
```

Comment:
Implementing the `LoggerAwareInterface` when the constructor already receives a logger doesn't look reasonable to me.

Reply:
I also wonder if we shouldn't change the other decorators to let them optionally receive a logger in their constructor too instead of implementing the `LoggerAwareInterface`

Reply:
doing so would also allow us to revert the `composer.json` changes